plugins/system/mvcoverride/language/en-GB/en-GB.plg_system_mvcoverride.ini

```diff
@@ -4,3 +4,9 @@ PLG_SYSTEM_MVC_OVERRIDE_MAKE_EXTENDABLE="Make Extendable"
 PLG_SYSTEM_MVC_OVERRIDE_MAKE_EXTENDABLE_DESC="Extend core component class. Its better to use OOP to extend components. You will be able to extend your override classes extend a Component[Controller|Model|View][Name]Default"
 PLG_SYSTEM_MVC_OVERRIDE_CHANGE_FUNCTIONS="Change functions"
 PLG_SYSTEM_MVC_OVERRIDE_CHANGE_FUNCTIONS_DESC="Changes all private methods in the base class to protected methods which allows the subclass to extend the base class better. This is not recommended unless you really need it."
+PLG_SYSTEM_MVC_OVERRIDE_PREFIX_LABEL="Prefix for extends classes"
+PLG_SYSTEM_MVC_OVERRIDE_PREFIX_DESC="Set prefix for exten classes. <br /><strong>Warning: at least one value must be set prefix or suffix or both</strong>"
```

Comment:
+PLG_SYSTEM_MVC_OVERRIDE_PREFIX_DESC="Set prefix for extend classes. <br /><strong>Warning: at least one value must be set: prefix, suffix or both</strong>"